script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -345,7 +344,7 @@ I'm stunned by her sudden rebuke.
 [sha:bcc6a8948d49f3f839f98bfb6ea1ce210985847f]{
 -- Page 69, Offset 1333.
 -- どうやらよほどの不興を買ったらしい。
-Apparently, I offended her.
+Apparently, I really offended her.
 }
 [sha:4ed6b353a9190036496ece060ba84f1516316584]{
 -- Page 70, Offset 1334.
```

Comment:
-- Page 70, Offset 1335.
I may have unintentionally said something that made people uncomfortable.

Reply:
I may have unintentionally said something that could make people uncomfortable.
// its not that he actually made the passerbys uncomfortable but he thought he might have said something that could have.

Reply:
You've already got the prevarication with 'may' but Petrikow's edit above makes it much stronger, the apparent offense, so this edit is a good suggestion to highlight Shiki's uncertainty.